scripts/build_energy_totals.py

```diff
@@ -669,31 +750,53 @@ def build_eurostat_co2(input_eurostat, countries, report_year, year=1990):
     return eurostat.multiply(specific_emissions).sum(axis=1)
 
 
-def build_co2_totals(countries, eea_co2, eurostat_co2):
+def build_co2_totals(countries, eea_co2, eurostat_co2, report_year):
     co2 = eea_co2.reindex(countries)
 
     for ct in pd.Index(countries).intersection(["BA", "RS", "AL", "ME", "MK"]):
-        mappings = {
-            "electricity": (
-                ct,
-                "+",
-                "Conventional Thermal Power Stations",
-                "of which From Coal",
-            ),
-            "residential non-elec": (ct, "+", "+", "Residential"),
-            "services non-elec": (ct, "+", "+", "Services"),
-            "road non-elec": (ct, "+", "+", "Road"),
-            "rail non-elec": (ct, "+", "+", "Rail"),
-            "domestic navigation": (ct, "+", "+", "Domestic Navigation"),
-            "international navigation": (ct, "-", "Bunkers"),
-            "domestic aviation": (ct, "+", "+", "Domestic aviation"),
-            "international aviation": (ct, "+", "+", "International aviation"),
-            # does not include industrial process emissions or fuel processing/refining
-            "industrial non-elec": (ct, "+", "Industry"),
-            # does not include non-energy emissions
-            "agriculture": (eurostat_co2.index.get_level_values(0) == ct)
-            & eurostat_co2.index.isin(["Agriculture / Forestry", "Fishing"], level=3),
-        }
+        if report_year != 2023:
```

Comment:
perhaps avoid the code duplication here? only adjust/overwrite differing keys in one if-clause

Reply:
Since now only the eurostat data from 2023 is used, the whole if-clause is obsolete